lib/install/examples/erb/hello_erb.js.erb

```diff
@@ -1,5 +1,6 @@
-// Run this example by adding <%= javascript_pack_tag 'hello_erb' %> to the head of your layout file,
-// like app/views/layouts/application.html.erb.
+// Run this example by adding <%%= javascript_pack_tag 'hello_erb' %> to the head of your layout file,
+// like app/views/layouts/application.html.erb. Please take care of removing the extra `%` from `<%%=`:
```

Comment:
IMO there is no need for that extra line?

Reply:
Which line do you mean? This one? `// we can't write the correct ERB tags here, they would get interpreted.`

Reply:
I was referring to `Please take care of removing the extra `%` from `<%%=`:
// we can't write the correct ERB tags here, they would get interpreted.`

I assume dev will just copy `<%%= javascript_pack_tag 'hello_erb' %>` and use it. But that's just an opinion. 

cc @gauravtiwari

Reply:
Ok, yes but `<%%= javascript_pack_tag 'hello_erb' %>` doesn't work as it is, you must use `<%= javascript_pack_tag 'hello_erb' %>`; it's a hint for the user.